src/main.rs

```diff
@@ -926,18 +926,26 @@ async fn main() {
     let opts = Opts::from_args();
     let dir = tempfile::tempdir().unwrap();
 
-    let (mut runner, layer) =
-        Runner::new_with_layer(opts.server, opts.token, dir.path().to_path_buf());
+    let (layer, jobs) = GitlabLayer::new();
 
-    let log_targets: filter::Targets = if let Some(log) = opts.log {
-        log.parse().unwrap()
+    let log_targets = if let Some(log) = opts.log {
+        log.parse::<filter::Targets>()
+            .unwrap()
+            .with_target("gitlab_runner::gitlab::job", Level::ERROR)
     } else {
         filter::Targets::new().with_default(Level::INFO)
     };
 
+    let fmt = tracing_subscriber::fmt::Layer::new();
+    let fmt = if std::io::stdout().is_terminal() {
+        fmt.pretty().boxed()
+    } else {
+        fmt.json().boxed()
```

Comment:
fwiw we just use a CLI option for this in the OBS runner (rather intentionally so since then the more readable pretty logs could be piped to a file during debugging)

Reply:
followed the lead there (mostly) :)